src/components/widgets/BaseInput.js

```diff
@@ -18,6 +19,7 @@ function BaseInput(props) {
       {...inputProps}
       className="form-control"
       readOnly={readonly}
+      autoFocus={autoFocus}
```

Comment:
I think we went with the convention of lowercasing html attribute names, as you can see with `readOnly -> readonly` above. Could you please rename `autoFocus` to `autofocus`?